server_cmds_test.go

```diff
@@ -306,19 +306,98 @@ func testCmdsServer(t *testing.T, c *Client) {
 
 		expected := []*OnlineClient{
 			{
-				ID:          5,
-				ChannelID:   7,
-				DatabaseID:  40,
-				Nickname:    "ScP",
-				Type:        0,
-				Away:        true,
-				AwayMessage: "not here",
+				ID:                             42087,
+				ChannelID:                      39,
+				DatabaseID:                     19,
+				Nickname:                       "bdeb1337",
+				Type:                           0,
+				Away:                           nil,
+				AwayMessage:                    nil,
```

Comment:
Did the default change here, not sure why?

Reply:
I tend to also think in your direction that it changed over time in some version. I might try to see if I can pinpoint that some way by running some queries on older versions perhaps.